src/main/resources/application.properties

```diff
@@ -7,7 +7,7 @@ spring.batch.job.enabled=true
 hub.username=sysadmin
 hub.password=blackduck
 hub.timeout=120000
-hub.server.url=http://eng-hub-verify01.dc1.lan:8080
+hub.server.url=http://eng-hub-appmgr-verify01.dc1.lan/
```

Comment:
Please do not checkin the new configuration files. The build might fail if it doesn't exist.